addons/sys_signal/fnc_handleSignalReturn.sqf

```diff
@@ -1,16 +1,14 @@
 /*
  * Author: ACRE2Team
- * SHORT DESCRIPTION
+ * Return function for signal calculation. Will be called after signal has been calculated in the extension
```

Comment:
Should be dot at the end, so it'll look nice when we generate the functions list (this one won't be there, but still, consistency).